app.py

```
@@ -0,0 +1,87 @@
+    # PRODUCTION: Fallback to simple health check API (FastAPI for consistency)
+    from fastapi import FastAPI
+    from fastapi.responses import JSONResponse
+
+    app = FastAPI(
+        title="SAMO AI API - Fallback Mode",
+        description="Fallback mode when main API is unavailable",
+        version="1.0.0"
+    )
+
+    @app.get("/health")
+    async def health():
```

Ruff (0.12.2)

Missing return type annotation for public function `health`

(ANN201)